pipelines/common-oci-ta.yaml

```diff
@@ -303,7 +303,7 @@ spec:
       - name: name
         value: clair-scan
       - name: bundle
-        value: quay.io/konflux-ci/tekton-catalog/task-clair-scan:0.2@sha256:076d5cde62b55bbfcdda2b4782392256bbda5ad38f839013b4330b3aba70a973
+        value: quay.io/konflux-ci/tekton-catalog/task-clair-scan:0.3@sha256:a7cc183967f89c4ac100d04ab8f81e54733beee60a0528208107c9a22d3c43af
```

Comment:
@clyang82 oci-ta does not need build-platforms like others, right?

Reply:
Yes. it is just one platform. do not need to add `matrix`.